src/screens/Sleep.tsx

```diff
@@ -61,6 +64,12 @@ export default function Sleep(): JSX.Element {
                 radioButtons = {radioButtons}
                 onPress = {onPressRadioButton}
             />
+
+                <View style={styles.sliderText}>
+                    <Text style={styles.text}>How was your sleep quality? (1-10): </Text>
+                    <Text style={styles.text}>{sliderValue}</Text>
+                </View>
+
```

Comment:
This block of code is indented weirdly and has random new lines at the top and bottom.
```suggestion
            <View style={styles.sliderText}>
                <Text style={styles.text}>How was your sleep quality? (1-10): </Text>
                <Text style={styles.text}>{sliderValue}</Text>
            </View>
```